settings.gradle

```diff
@@ -33,8 +25,9 @@ dependencyResolutionManagement {
             library("errorProne-core", "com.google.errorprone:error_prone_core:2.18.0")
             library("errorProne-javac", "com.google.errorprone:javac:9+181-r4173-1")
             library("guava", "com.google.guava:guava:31.1-jre")
-            library("jspecify", "org.jspecify:jspecify:0.0.0-SNAPSHOT")
+            library("jspecify", "org.jspecify:jspecify:0.3.0")
```

Comment:
I'm a bit confused by the different versions used... `demo` uses `jspecify-0.0.0-SNAPSHOT.jar` and the conformance artifacts below use `0.0.0-SNAPSHOT`. Why is this now 0.3.0?

Reply:
I'm going to document this somewhere in the repo as well, but here's what I'm thinking:

The reference checker has the following dependencies:

- on `jspecify` for its implementation: some of its own APIs are annotated
- on the conformance test suite ZIP file
- on the conformance test framework library

Those would also be dependencies of any JSpecify-compliant tool, although the first one is optional: they don't have to annotate their own code.

For each of those dependencies, I want developers to be able to depend on either a published version (snapshot or not) or a local build.

I expect the version numbers of those three things, once they're published, to vary independently. We'd update the test suite version every time we change or add a test case; the test framework library only when that changes; and jspecify itself very rarely.

I want to publish snapshot versions of the test suite and the test framework so that by default the reference checker depends on those for its tests. That puts the reference checker on the same footing as any other tool, which I would expect to use (probably non-snapshot) published versions of both.

So the idea will be, eventually, that by default the reference checker uses published snapshots of both, but there's a simple way to use a local clone (at whatever commit) for either one. Then our CI would use a local clone of the `main` branch for both in order to retain the current behavior.

That's what I'm trying to do now (but I'm not done) for the conformance test suite and `jspecify`: If you want the local build, you `--include-build path/to/jspecify` and those get substituted no matter what version is specified here. Otherwise, we'll use `jspecify:0.3.0` (because why shouldn't we?) and `conformance-tests-0.0.0-SNAPSHOT` (which is [published](https://s01.oss.sonatype.org/content/repositories/snapshots/org/jspecify/conformance/conformance-tests/0.0.0-SNAPSHOT/) already, although only manually for now).